src/odd.js

```diff
@@ -60,64 +62,74 @@ async function signup(odd, username) {
 
   const content = new TextDecoder().decode(await fs.read(profileFilePath))
   console.log("profile data :", content)
+
+  const timeout = setTimeout(() => {
+    clearTimeout(timeout)
+    window.location.href = "/app";
+  }, 5000)
```

Comment:
this admittedly isn't ideal, but since we're forcing a hard refresh on the page, we need to wait for the DNS links to be updated. in an SPA, the code would continue executing during a route change, but this isn't the case with how this app is currently setup. there are some different options that could be explored